packages/embed/src/setups/version4.js

```diff
@@ -4,11 +4,11 @@ import loadAsset from './loadAsset';
 export default async function setupVersionFamily4(
   { assets },
   { botIconURL, directLineURL: domain, userId, webSocket },
-  { language, secret, token, username }
+  { language, secret, token, username, conversationStartLocale }
```

Comment:
This is more than just passing a parameter.

Today:

- We read the `&l=XXX` parameter, and then pass it to Web Chat UI (not DLJS)
   - Web Chat will:
      - Passthrough the locale to the bot (per activity), even if it is invalid
      - Normalize the language by guessing an invalid locale, e.g. `123` -> `en`, `zh-TW` -> `zh-Hant-TW`
      - Web Chat use the normalized version of UI locale
- Embed support two types of locale:
   - Azure locale, for example, `zh-hant.zh-hk` (this is using Web Chat inside Azure Portal)
      - When receiving `zh-hant.zh-hk`, the embed will pass `zh-Hant-HK` to Web Chat instead (ISO format)
      - The second part is the culture. For example, `zh-hant.zh-hk` will display UI in Traditional Chinese, and currency/datetime in Hong Kong format
   - ISO format locale, for example, `en` and `zh-Hant-HK`

Please ask @p-nagpal how to handle Azure locale:

1. Should we convert Azure locale to ISO locale before sending to the bot?
1. Should we passthrough Azure locale to the bot?